components/kyma-environment-broker/internal/storage/dbsession/factory.go

```diff
@@ -45,6 +47,8 @@ type WriteSession interface {
 	UpdateOperation(instance dbmodel.OperationDTO) dberr.Error
 	InsertOrchestration(o internal.Orchestration) dberr.Error
 	UpdateOrchestration(o internal.Orchestration) dberr.Error
+	InsertRuntimeState(state dbmodel.RuntimeStateDTO) dberr.Error
+	UpdateRuntimeState(state dbmodel.RuntimeStateDTO) dberr.Error
```

Comment:
we do not need a runtime step. every time something is changed - new entry is created